widgetry/src/widgets/spinner.rs

```diff
@@ -1,21 +1,46 @@
 use geom::{CornerRadii, Distance, Polygon, Pt2D};
 
 use crate::{
-    include_labeled_bytes, text, Button, Drawable, EdgeInsets, EventCtx, GeomBatch, GfxCtx,
-    Outcome, OutlineStyle, Prerender, ScreenDims, ScreenPt, ScreenRectangle, Style, Text, Widget,
+    include_labeled_bytes, Button, Drawable, EdgeInsets, EventCtx, GeomBatch, GfxCtx, Outcome,
+    OutlineStyle, Prerender, ScreenDims, ScreenPt, ScreenRectangle, Style, Text, Widget,
     WidgetImpl, WidgetOutput,
 };
 
-// TODO MAX_CHAR_WIDTH is a hardcoded nonsense value
-const TEXT_WIDTH: f64 = 2.0 * text::MAX_CHAR_WIDTH;
+// Manually tuned
+const TEXT_WIDTH: f64 = 80.0;
+
+pub trait SpinnerValue:
```

Comment:
Thanks to https://www.worthe-it.co.za/blog/2017-01-15-aliasing-traits-in-rust.html